client/components/theme-collection/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { Button } from '@wordpress/components';
+import { Icon, chevronLeft, chevronRight } from '@wordpress/icons';
+import { ReactElement, useEffect, useRef } from 'react';
+import './style.scss';
+import Swiper from 'swiper';
+import { Navigation, Keyboard, Mousewheel } from 'swiper/modules';
+import { ThemeBlock } from 'calypso/components/themes-list';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/keyboard';
+import 'swiper/css/mousewheel';
+
+interface ThemeCollectionProps {
```

Comment:
This interface needs to be refactored. I've duplicated the props in the PremiumCollection for now.

We'll need a generic interface that this interface will extend.